<launch>
  <!-- <arg name="folder" value="$(find so_dso_loop_closure)/loop_closure/results/KITTI/06" /> -->
  <!-- <arg name="folder" value="$(find so_dso_loop_closure)/loop_closure/results/Tara/grass" /> -->

  <!-- <arg name="date" value="2014-07-14-14-49-50" /> -->
  <!-- <arg name="date" value="2014-11-28-12-07-13" /> -->
  <!-- <arg name="date" value="2014-12-12-10-45-15" /> -->
  <!-- <arg name="date" value="2015-02-10-11-58-05" /> -->
  <arg name="date" value="2015-05-19-14-06-38" />
  <!-- <arg name="date" value="2015-05-22-11-14-30" /> -->
  <!-- <arg name="date" value="2015-08-13-16-02-58" /> -->
  <!-- <arg name="date" value="2015-10-30-13-52-14" /> -->

  <arg name="folder" value="$(find so_dso_loop_closure)/loop_closure/results/RobotCar/$(arg date)" />

  <node name="test_sc" pkg="so_dso_loop_closure" type="test_sc" output="screen" required="true">
    <param name="poses_history_file" value="$(arg folder)/poses_history_file.txt" />
    <param name="pts_history_file" value="$(arg folder)/pts_history_file.txt" />
    <param name="sc_file" value="$(arg folder)/historySC.txt" />
    <param name="incoming_id_file" value="$(arg folder)/incoming_id_file.txt" />
    <param name="loopRange" value="45.0" />
    <param name="voxelAngle" value="1.0" />
    <!-- <param name="loopRange" value="2.0" />
    <param name="voxelAngle" value="2.0" /> -->
  </node>
</launch>
